assets/Discord-ServiceContract.md

```diff
@@ -0,0 +1,43 @@
+# Discord
+
+## Service Manager: _Chris Daniel_
+### Lifecycle: _Live_
+### Evolution: _Product_
+
+## Service Description
+_Discord is the easiest way to talk over voice, video, and text. Talk, chat, hang out, and stay close with other members of the Wardley Mapping Community._
+
+_Discord, operated by the foundation, will gather community members in one place and boost interaction and collaboration._
+
+_To create your account, follow [this link](https://discord.com/invite/upg2deYD3Z) and let the Discord app guide you._
+
+_If you need a new community channel, request one by opening issue in [this Github repository](https://github.com/Wardley-Mapping-Foundation/wmf-operating-model)._
+
+_When any assistance is needed, notify @admin or @moderator in a chat._
+
+
+## User Needs
```

Comment:
Per @juliusgb's map (will deeplink to Miro on Discord) there was another user need:

As a server admin
I need to run, and moderate, and grow community
in order to create a safe community that upholds the principles (or values, wording?) of the Wardley Mapping Foundation

or maybe expressed as community member user needs?

Reply:
@agvbergin, @juliusgb : I don't think this is a user's need... it looks like an internal requirement regarding how we run the service.  Those don't need to be listed in the contract.